### DISCUSSION TITLE
filepath error HW 01 Part 3

### DISCUSSION THREAD

**Discussion by saewon1** [cooperrc/computational-mechanics#81] [Q&A] [answered]
When running the code cell 

```python
import numpy as np
year, pop = np.loadtxt('../data/world_population_1900-2020.csv',skiprows=1,delimiter=',',unpack=True)
print('years=',year)
print('population =', pop)
```

for the module 1 HW, I keep getting the error "FileNotFoundError: [Errno 2] No such file or directory"

**Comment by cooperrc** [comment]
> When running the code cell
> 
> ```python
> import numpy as np
> year, pop = np.loadtxt('../data/world_population_1900-2020.csv',skiprows=1,delimiter=',',unpack=True)
> print('years=',year)
> print('population =', pop)
> ```
> 
> for the module 1 HW, I keep getting the error "FileNotFoundError: [Errno 2] No such file or directory"

This is good because it means your `HW_01.md` file is saved in your `work` folder :+1: 

Its bad, because we can't find the `data` folder for `world_population`. 

You can solve this a few ways, here is the quickest:
```python
year, pop = np.loadtxt('~/computational-mechanics/data/world_population_1900-2020.csv',
skiprows=1,
delimiter=','
unpack=True)
```

I am changing the file location to home folder [`~`] inside that, `computational-mechanics`, then go into `data` to get the file.

**Reply by saewon1** [reply]
I tried the new code:

``` python
year, pop = np.loadtxt('~/computational-mechanics/data/world_population_1900-2020.csv',
skiprows=1,
delimiter=',', 
unpack=True)
#comma added after "delimeter = ','"
```

But there is now a new error: "OSError: ~/computational-mechanics/data/world_population_1900-2020.csv not found."

**Comment by saewon1** [comment, marked as answer]
in the end I made a copy of the data file and moved it to the home location? I also changed the name to be more simple. I am not sure but I used the move command and moved it to the blank filepath. Then I input the filepath I saw under the move command ("/home/joyvan"), and this is the code that worked:

``` python
import numpy as np
year, pop = np.loadtxt('/home/jovyan/worldpop.csv',
skiprows=1,
delimiter=',',
unpack=True)
print('years=',year)
print('population =', pop)
```

I dont know what jovyan is but apparently there is another file after the home file.
